docker-swarmkit/snapcraft.yaml

```diff
@@ -0,0 +1,27 @@
+name: docker-swarmkit
+version: master
+summary: A toolkit for orchestrating distributed systems at any scale
+description: |
+  SwarmKit is a toolkit for orchestrating distributed systems at any scale. It
+  includes primitives for node discovery, raft-based consensus, task scheduling
+  and more.
+confinement: devmode
+
+apps:
+  swarmd:
+    command: bin/swarmd
```

Comment:
I think this shouldn't be an autostarted daemon, because the ca-example shows how to run it with passing some arguments. But I'm not sure.